pants_plugins/sendwave/pants_docker/package.py

```diff
@@ -133,19 +153,18 @@ async def package_into_image(
     docstring for more information)
     """
     target_name = field_set.address.target_name
+    logger.debug("Building Target %s", target_name)
```

Comment:
do we need this log line? I thought something like this already gets logged?

Reply:
This log line already existed in the code, I just moved it up to the top of this function definition since now we start iterating the targets much sooner! I'm open to dropping it though, since we're also logging the build command below.

https://github.com/waveremit/pants-docker/blob/e20083ffc4445f105fdf0ad0bef7aab4aa64deb0/pants_plugins/sendwave/pants_docker/package.py#L142